ddp-workspace/projects/toolkit/tsconfig.lib.json

```diff
@@ -14,7 +14,7 @@
     "types": [],
     "lib": [
       "dom",
-      "es2018"
+      "es2019"
```

Comment:
Presumably for flatMap? Make sure you can `ng build` the pancan project.

Reply:
Correct! Yes, it works w/o errors :)